examples/notebooks/python/mortgage-gpu.ipynb

```diff
@@ -112,8 +112,8 @@
     "])\n",
     "features = [ x.name for x in schema if x.name != label ]\n",
     "\n",
-    "train_data = reader.schema(schema).option('header', True).csv('/data/mortgage/csv/train')\n",
-    "trans_data = reader.schema(schema).option('header', True).csv('/data/mortgage/csv/test')"
+    "train_data = reader.schema(schema).option('header', True).csv('/data/mortgage/csv/small-train.csv')\n",
```

Comment:
Also, I don't think we need to update this file.
Nothing changed but output.

Reply:
restored